frontend/src/views/HomeView.vue

```diff
@@ -77,11 +82,16 @@ export default {
         }
 
         await Promise.all([fetchTasks(), fetchTeams()])
```

Comment:
[Try!] await しているから例外が普通に飛んできます。以下のように同期同様に書けますよ。

```suggestion
    try {
        await Promise.all([fetchTasks(), fetchTeams()])
    } catch (err) {
        /// ... エラーハンドリング
    }     
```